presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -198,6 +199,16 @@ public static Slice stCentroid(@SqlType(GEOMETRY_TYPE_NAME) Slice input)
         return serialize(createFromEsriGeometry(centroid, geometry.getEsriSpatialReference()));
     }
 
+    @Description("Returns the minimum convex geometry that encloses all geometries within the set")
+    @ScalarFunction("ST_ConvexNull")
+    @SqlType(GEOMETRY_TYPE_NAME)
+    public static Slice stConvexNull(@SqlType(GEOMETRY_TYPE_NAME) Slice input)
+    {
+        OGCGeometry geometry = deserialize(input);
+        validateType("ST_IsConvexNull", geometry, EnumSet.of(MULTI_LINE_STRING, MULTI_POINT, MULTI_POLYGON, GEOMETRY_COLLECTION));
```

Comment:
The SQL/MM Part 3 standard defines ST_ConvexHull function for all types of geometries, not only multi types and geometry collections. Hence, this check is not needed.